### PR TITLE
Setup linters

### PR DESCRIPTION
**Pull request by darrenodi**
In this branch, I setup linters for a project I did in the past.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            <div class="pricing">
+                <div class="first-offer">
+                    <div class="offer-title">
+                      TENOR TROMBONE
+                    </div>
+                   <h2>$600</h2>
+                       <p>Lorem ipsum.</p>
+                       <p>Lorem ipsum.</p>
+                       <p>Lorem ipsum dolor.</p>
+                       <p>Lorem ipsum.</p>
```
[the remark this answers is not in the document]
Comment (descholar-ceo) [reply index.html:94]: :+1: Great comment :heavy_check_mark:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                        <button type="button" class="offer-button">SELECT</button>
```
[the remark this answers is not in the document]
Comment (descholar-ceo) [reply index.html:95]: :+1: Great comment :heavy_check_mark:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                </div>
+                <div class="first-offer">
+                    <div class="offer-title">
+                      BASS TROMBONE
+                    </div>
+                    <h2>$900</h2>
+                    <p>Lorem ipsum.</p>
+                    <p>Lorem ipsum.</p>
+                    <p>Lorem ipsum dolor.</p>
+                    <p>Lorem ipsum.</p>
```
[the remark this answers is not in the document]
Comment (descholar-ceo) [reply index.html:105]: :+1: Great comment :heavy_check_mark:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                </div>
+                <div class="first-offer">
+                    <div class="offer-title">
+                      VALVE TROMBONE
+                    </div>
+                    <h2>$1200</h2>
+                    <p>Lorem ipsum.</p>
+                    <p>Lorem ipsum.</p>
+                    <p>Lorem ipsum dolor.</p>
+                    <p>Lorem ipsum.</p>
```
[the remark this answers is not in the document]
Comment (descholar-ceo) [reply index.html:116]: :+1: Great comment :heavy_check_mark:

File: index.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+                    <button type="button" class="offer-button">SELECT</button>
+                </div>
+            </div>
+        </div>
+    </main>
+    <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js" charset="utf-8"></script> -->
```
[the remark this answers is not in the document]
Comment (descholar-ceo) [reply index.html:122]: - [ ] I would suggest you explain why you requested this change

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+  max-width: 150px;
+  width: 100%;
+  height: 30px;
+  margin: 15px 0;
+  border: 0;
+  background-color: #f1c40f;
+}
+
+/* @media (max-width: 428px) {
+} */
```
[the remark this answers is not in the document]
Comment (descholar-ceo) [reply styles.css:108]: :+1: Great comment :heavy_check_mark:

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @darrenodi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [x] I would suggest you always to explain why you requested a given changes request

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @darrenodi. 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :male_detective: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by darrenodi** [comment]
Thank you :)

**Pull request merged**